LibNFS.xs

```diff
@@ -983,6 +983,50 @@ static void _set_unix_authn(pTHX_ struct nfs_context* nfs, SV* value_sv) {
     nfs_set_auth(nfs, auth);
 }
 
+#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
+static void _set_readdir_max_buffer_size(pTHX_ struct nfs_context* nfs, SV* value_sv) {
+
+    uint32_t dircount, maxcount;
+
+    if (SvROK(value_sv) && (SvTYPE(SvRV(value_sv)) == SVt_PVAV)) {
+        #define MAX_BUFFER_SIZE_ARRAY_SIZE 2
```

Comment:
This should IMO be outside the function, but I can do that in a follow-up commit.